_data/mentors.yml

```diff
@@ -45,3 +45,10 @@
   twitter: jigyasa_grover
   facebook: jigyasa.grover.14
   bio:
+
+- name: Shubham Padia
+  github: shubham-padia
+  image: shubham-padia.jpg
+  twitter: shubham_p98
+  facebook: padiashubham
+  bio:
```

Comment:
Line contains following spacing inconsistencies:
- No newline at EOF.

*SpaceConsistencyBear, severity NORMAL, section `default`.*

The issue can be fixed by applying the following patch:

```diff
--- a/_data/mentors.yml
+++ b/_data/mentors.yml
@@ -51,4 +51,4 @@
   image: shubham-padia.jpg
   twitter: shubham_p98
   facebook: padiashubham
-  bio:+  bio:
```